File: payroll-api/src/main/java/com/company/payroll/model/Salary.java
package com.company.payroll.model;

import java.math.BigDecimal;
import java.time.LocalDate;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@RequiredArgsConstructor
public class Salary {
    private Integer sId;

    private BigDecimal monthlysalary;

    private BigDecimal annualsalary;

    private LocalDate dateupdate;

}

File: kotlin-payroll/src/main/kotlin/com/company/payroll/model/Salary.kt
package com.company.payroll.model

import lombok.AllArgsConstructor
import lombok.RequiredArgsConstructor
import java.math.BigDecimal
import java.time.LocalDate

@AllArgsConstructor
@RequiredArgsConstructor
data class Salary(
    var sId: Int,
    var monthlySalary: BigDecimal,
    var annualSalary: BigDecimal,
    var dateUpdate: LocalDate
)
